src/components/AssetAccounts/AssetAccountRow.tsx

```diff
@@ -16,8 +16,8 @@ import { Allocations } from 'components/AccountRow/Allocations'
 import { Amount } from 'components/Amount/Amount'
 import { AssetIcon } from 'components/AssetIcon'
 import { RawText } from 'components/Text'
-import { middleEllipsis } from 'lib/utils'
-import { accountIdToFeeAssetId, accountIdToLabel } from 'state/slices/portfolioSlice/utils'
+import { accountIdToFeeAssetId, middleEllipsis } from 'lib/utils'
```

Comment:
Was this moved because of circular dependencies ? While I agree to move this out of portfolioSlice, it may make more sense to have this live in an `accounts` domain?

Reply:
Correct, moved out cue to circular imports. No probs, will move this into an accounts util along with other pre-existing helpers in that file 👍